### ISSUE TITLE
Add the rad_max data in the `observation.create`

### ISSUE THREAD

**Issue by bkhelifi** [gammapy/gammapy#4060]
**Is your feature request related to a problem? Please describe.**
Associated to the discussion #4059, the rad_max IRF is not yet stored when it exists within the function `observation.create`.

**Describe the solution you'd like**
If we have point_like IRFs, one should return indeed:

```
        return cls(
            obs_id=obs_id,
            obs_info=obs_info,
            gti=gti,
            aeff=irfs.get("aeff"),
            bkg=irfs.get("bkg"),
            edisp=irfs.get("edisp"),
            psf=irfs.get("psf"),
            rad_max=irfs.get("rad_max") ,
        )
```

**Comment by registerrier** [comment]
Note that this is corrected by #4024

**Comment by bkhelifi** [comment]
One comment: from the init function, rad_max is set at the same level than the IRFs, whereas it is NOT an IRF (see the IRF theory, the GADF...).
rad_mad is a meta data associated to the (MC) IRFs. This is strange that we pass it in constructor... This is maybe a bias of the used MAGIC DL3 file.
I think that this behaviour and coding should be changed for the v1.1: rad_max should be kept as meta data, a good test file should be associated to that, whereas maybe keeping this backward compatibility for the MAGIC DL3 production...
